src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx

```diff
@@ -3,6 +3,20 @@
 Amplify requires a minimum deployment target of 13.0 and Xcode 13.2 or higher when targeting iOS.
 
 From your project root, navigate to the `ios/` directory and open the `Podfile` in a text editor of your choice. At the top of the file, update the target iOS platform to 13.0 or higher.
+## Web
+
+### Auth category
+
+No specific configuration steps are needed to begin using the Auth category in the browser.
+
+### Analytics and Storage categories
+
+import flutter2 from '/src/fragments/lib/project-setup/flutter/platform-setup/sqlite3_wasm.mdx';
+
+<Fragments fragments={{ flutter: flutter2 }} />
```

Comment:
This is no longer needed. We no longer use sqlite on web.

I think you can get rid of the Auth section above, and just update the Web section to say

> No specific configuration steps are needed to begin using Amplify in the browser.